_data/monthly-calls.yml

```diff
@@ -0,0 +1,88 @@
+videos:
+  - title: Conducting user research with people with disabilities
+    subtitle: How to develop a sustainable, repeatable research process
+    description: "One of our roadmap goals is to conduct more user research with people who have disabilities — and to ensure we’re doing so regularly. We shared our progress and reported on findings from our first round of this research: working with users who have visual impairments to conduct usability tests on Design System components."
```

Comment:
Use lowercase for "Design System" here

Reply:
Updated in [43b65e9](https://github.com/uswds/uswds-site/pull/2345/commits/43b65e9e9d18fe84aa856ca94d589da9528efba7)